cloud/upgrades.rst

```diff
@@ -0,0 +1,20 @@
+Upgrades
+########
+
+On a self-hosted instance of Citus, all upgrades to PostgreSQL and/or the Citus extension must be done :ref:`manually <upgrading>`.  However on Citus Cloud the work is automated. When desired, visit the Settings tab in the cloud console, and scroll down to the upgrade section:
```

Comment:
Could we take out the part about 'self-hosted Citus needs manual upgrade?". Maybe instead just replacing it with spiel about Citus Cloud (Citus Cloud allows you to upgrade to the latest versions of Citus and PostgreSQL with a few clicks of the button, etc".  Also maybe something like "we recommend upgrading to the latest versions so you get all the new features and bug-fixes etc".